build.gradle

```diff
@@ -35,7 +35,7 @@ ext {
 
 ext {
     // libs
-    wordpressLintVersion = '1.1.0'
+    wordpressLintVersion = '2.0.0'
```

Comment:
Just a question: why don't we need to exclude `generated` files, like we did in `WordPress-Android` repository? https://github.com/wordpress-mobile/WordPress-Android/pull/19434/files#diff-e8b41fbc109de9b7fa37c43933345e6050e96b3c94fd279696d2adbfb9a90d38R25

Reply:
Good question @wzieba ! 💯 

Please follow [this](https://github.com/wordpress-mobile/WordPress-Android/pull/19434#discussion_r1371495113) response comment of mine to understand the why. 🙏

Reply:
Oh gotcha, thanks!